src/abstract-interpretation/data-frame/domain.ts

```diff
@@ -60,6 +69,10 @@ export function subtractColNames(X1: ColNamesDomain, X2: ColNamesDomain): ColNam
 	}
```

Comment:
regarding subtract col names: why does `x - TOP` return Bottom?

Reply:
but isn't TOP equivalent to "we do not know which column names are removed"?

If i do `select(-contains(u))` to remove all columns that contain an unknown string, then the resulting dataframe might be empty (which woudln't be bot in our definition but an empty list right?) or it could contain elements - but it would not be invalid